src/overmapbuffer.cpp

```diff
@@ -1722,3 +1723,88 @@ overmapbuffer::electric_grid_connectivity_at( const tripoint_abs_omt &p )
 
     return ret;
 }
+
+bool overmapbuffer::add_grid_connection( const tripoint_abs_omt &lhs, const tripoint_abs_omt &rhs )
+{
+    if( project_to<coords::om>( lhs ) != project_to<coords::om>( rhs ) ) {
```

Comment:
It doesn't let me connect omts on different z-levels (e.g. house room and house roof) - is this intentional?
If not, this fixes it.
```suggestion
    if( project_to<coords::om>( lhs ).xy() != project_to<coords::om>( rhs ).xy() ) {
```